pkg/asset/installconfig/azure/validation.go

```diff
@@ -146,11 +147,46 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 		}
 	}
 
-	// The UltraSSDAvailable capability might not be present at all, in which case it must assumed to be false
-	val, ok = capabilities["UltraSSDAvailable"]
-	if ok {
-		ultraSSDAvailable := strings.EqualFold(val, "True")
-		if ultraSSDEnabled && !ultraSSDAvailable {
+	if ultraSSDEnabled {
+		locationInfo, err := client.GetLocationInfo(context.TODO(), region, instanceType)
+		if err != nil {
+			errMsg := fmt.Sprintf("could not determined Availability Zones support in the %s region: %v\n", region, err)
```

Comment:
errMsg := fmt.Sprintf("could not determine Availability Zones support in the %s region: %v\n", region, err)

Reply:
Done